src/hooks/hooks/handlers/BigOrderIsProfitHookHandler.ts

```diff
@@ -0,0 +1,139 @@
+import BigNumber from "bignumber.js";
+import Web3 from "web3";
+
+import { createClientLogger } from "../../../logger";
+import { Notification } from "../../notification/Notification";
+import { TwitterNotification } from "../../notification/TwitterNotification";
+import { OrderEstimatedParams } from "../../types/params/OrderEstimatedParams";
+import { Hook } from "../Hook";
+
+export const hookHandlerBigOrderIsProfit = (
+  consumerKey: string,
+  consumerSecret: string,
+  accessTokenKey: string,
+  accessTokenSecret: string,
```

Comment:
What does all these args mean? When you look at `hookHandlerBigOrderIsProfit()`, there is nothing that reminds us about twitter credentials, so you need to give proper names.

Also, it seems strange that `hookHandlerBigOrderIsProfit` is only suitable for Twitter: you can't reuse it for Telegram. Since you ALREADY introduced a `Notification` interface, why don't you introduce smth like this:

``` 
function hookHandlerBigOrderIsProfit(notifier: Notification, minUsdAmount: number)
```
?